### DISCUSSION TITLE
System.Text.Json vulnerability

### DISCUSSION THREAD

**Discussion by martin-honnen** [ikvmnet/ikvm#593] [General]
https://github.com/advisories/GHSA-hh2w-p6rv-4g7w gives a vulnerability for System.Text.Json 8.0.0 as used in IKVM 8.10.1 via https://www.nuget.org/packages/Microsoft.Extensions.DependencyModel/8.0.0.
It seems a fix with System.Text.Json 8.0.4 is in the 8.0.1 version of https://www.nuget.org/packages/Microsoft.Extensions.DependencyModel.
Is there any way for a user of IKVM to use the current IKVM version and ensure it uses the 8.0.1 fix? 
Or does that require a new IKVM release?

**Comment by wasabii** [comment]
NuGet unifies to the highest dependent package version. So just install the later one?
